src/controller/ApplicantProfileController.ts

```diff
@@ -0,0 +1,44 @@
+import { NextFunction, Request, Response } from "express"
```

Comment:
!This comment about all files inside the project.
add prettier settings to the project and format with prettier all files